email-log.php

```diff
@@ -77,3 +77,5 @@ function email_log_deactivate() {
 
 // Fix compatibility issues with wpmandrill plugin.
 require_once plugin_dir_path( __FILE__ ) . 'include/compatibility/wpmandrill.php';
+
+require_once  __DIR__ . '/vendor/collizo4sky/persist-admin-notices-dismissal/persist-admin-notices-dismissal.php';
```

Comment:
You don't need to include the file like this. You need to register the namespace. Check https://github.com/sudar/email-log/blob/dev/2.4.0/load-email-log.php#L29

Reply:
Fixed.